{"uuid":"cb4d5818-65ba-416f-915d-82af3e02f8cf","historyId":"4e92c9596c36fcad46b7a804e4236d95","fullName":"com.qacart.todo.testcases.TodoTest.shouldBeAbleToDeleteTodo","labels":[{"name":"package","value":"com.qacart.todo.testcases.TodoTest"},{"name":"testClass","value":"com.qacart.todo.testcases.TodoTest"},{"name":"testMethod","value":"shouldBeAbleToDeleteTodo"},{"name":"parentSuite","value":"Surefire suite"},{"name":"suite","value":"Surefire test"},{"name":"subSuite","value":"com.qacart.todo.testcases.TodoTest"},{"name":"host","value":"DESKTOP-BKFTL5O"},{"name":"thread","value":"22044@DESKTOP-BKFTL5O.TestNG-test-Surefire test-3(29)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"story","value":"Delete todo"},{"name":"feature","value":"Todo Management"}],"links":[],"name":"Should be able to delete todo","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Something went wrong in adding the todo. Status code: 400","trace":"java.lang.RuntimeException: Something went wrong in adding the todo. Status code: 400\r\n\tat com.qacart.todo.api.TaskApi.addTask(TaskApi.java:27)\r\n\tat com.qacart.todo.testcases.TodoTest.shouldBeAbleToDeleteTodo(TodoTest.java:67)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\r\n\tat java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)\r\n\tat java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)\r\n\tat org.testng.internal.thread.graph.TestNGFutureTask.run(TestNGFutureTask.java:22)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)\r\n\tat java.base/java.lang.Thread.run(Thread.java:834)\r\n"},"stage":"finished","description":"This test verifies that a user can delete a todo item successfully.","steps":[],"attachments":[],"parameters":[],"start":1724821739573,"stop":1724821742833}